graphsdk/src/androidTest/java/com/microsoft/graph/http/UrlConnectionTests.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.graph.http;
+
+import android.test.AndroidTestCase;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test cases for {@see UrlConnection}
+ */
+public class UrlConnectionTests extends AndroidTestCase {
```

Comment:
Hi Peter, Are you sure we need to send a true http request for this unit test? We think the MockConnection should be sufficient here.